docs/src/image_loading.md

```diff
@@ -0,0 +1,143 @@
+- Loading SOC images to target components. The SOC images could come from a [Flash Storage](./flash_layout.md) or from another platform (e.g. BMC).
+- Retrieving the SOC Image Metadata (as defined in the [SOC Manifest](https://github.com/chipsalliance/caliptra-sw/blob/main-2.x/auth-manifest/README.md) documentation)
+- Verifying and authenticating the SOC Images.
+
+The diagram below provides an **example** of how the Caliptra subsystem, integrated with custom SOC elements (highlighted in green), facilitates the loading of SOC images to vendor components.
+
+Custom SOC elements:
+
+* **External Flash** : A flash storage containing SOC manifest and the SOC images.
+* **BMC** : An external Board Mangement Controller that provides a service to source the SOC images and manifest.
+* **Vendor CPU**: A custom CPU which executes code from a coupled Vendor RAM
+* **Vendor RAM**: RAM exclusively used by the Vendor CPU and is programmable via AXI bus.
+* **Vendor Cfg Storage**: A volatile memory storage used to contain vendor specific configurations.
+* **SOC Images** SOC Image 1 is a firmware for Vendor CPU and loaded to Vendor RAM. SOC Image 2 is a configuration binary to be loaded to Vendor Cfg Storage.
+* **SOC Config** : A register accessible by the MCU ROM to select appropriate source (flash or BMC) for loading the SOC images.
+* **Caliptra 'Go' Wire** : A signal controlled by the Caliptra core routed to the reset line of the Vendor CPU.
+
+<p align="center">
+    <img src="images/image_loading_sample.svg" alt="flash_config" width="80%">
+</p>
+
+## Image Loading Steps
+
+The sequence diagram below shows the high level steps of loading MCU RT image and SOC images.
+
+* *Red Arrows indicates actions taken by Caliptra RT*
+* *Purple Arrows indicates actions taken by MCU ROM*
+* *Blue Arrows indicates actions taken by MCU RT*
+* *Black Arrows indicates actions taken by BMC FW*
+
+<p align="center">
+    <!--- https://www.plantuml.com/plantuml/uml/bLLlR_es47xlJt6_JffQgONrJpU95rDfrZ-LXfY0TPF6L9da0fQ2ZPnBM_hfvnmuYHlA-BsAxNly-EwvnyvDJj9IiSp4DrhW4Y5JymNDhLoBbGkeMAsa9ZWQ3oE8Z4xLlB2IbD578FFjIeYCPAPMP2K3_EGJsMCV-d7C23VUlHy6KcKpyrwY_3X43g8_EUY-hG2CH_s1ZxwKan0omjYyebt3K_UUONuXXDqE8ewwRnYupsI-u12Fmk0pwiHOfyvoPJHgObpiPq9eGmZbQPK-FHWLMYizhrPWD1o8BjHGLqUDSfCMe3gW1tVl61TbbzoYHN1VYpcIa0KPNIndQCMcTMV__60n-VVNEe-dfPmZZ4rXO_V136EQwcCkikirEX6xQZXbK0_w1IsCLHy8KYVmkv59ewW9tFzrOFqC5GA4QEGDEPcLMyY8Uz88N7JEE-V2R_U5-PJT0IYJDQJ6GkdbWTGgnPo4OpchKM7CBtkohGj3L2ivp7BSk29CpoKeUFaxwNGI270fMaNw-KPxrAHYIHZG7gRXrS4QiW4d6xTyqZ28jJJqrp647IZYLI3YuNnj5Q-5y3izatk9Ul3egC6_7sAFr7oxfGrLXgbcWjSSjcyS4-vX6IF1tTztKVTX2BUAMaY-zGy6tfH5o97Ah8FORqWO4oQm8G2Lxu3nEQoClzEWj4jD9yCq1PAp3CM-bsOYMeGZ9Alm5V-7jRPPm7_gLigEJVu0T1t1pJ-bsNNVv5bMOBFB3rKX0oIPI98GcSmTpWJSPY4oOrPiKoojnorIR0kVpBULbVEb14Bp0MIBbeVvY6wKpci3iq4QWH-X3jLlRmFmPJbDkBtiDBPKWGhAXrxyr0HnbKuHZwaV4dlL0UcVtsCk-KKb9tlyPwyxy8nMfUkDzPMkd3lLRueMWRyPCzKihMVVH_vpnvkxwKqFs97_BvtIM8NLa1TnZ7akjh7Gp3vMQx3pIXa-rQHGoJhLiR4MSpTBIdUJ-RgEQbDu0L4dKFfBtBXHiSp-0m00 -->
```

Comment:
gotcha. Yeah I can remove SOC Configuration and just mention it. Yes technically, Image Metadata Collection(IMC) is inside the SOC Manifest which PLDM transfers as 1 component. SOC Manifest is basically composed of a preamble+IMC, but for flash, we can skip through the preamble part and just retrieve directly the Image Metadata Collection. I think to avoid confusion, I can just say Get Image Data Collection for both and just mention in the step that it is part of the SOC Manifest.